test/extended/images/mysql_replica.go

```diff
@@ -0,0 +1,148 @@
+package images
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+var _ = g.Describe("images: MySQL replication template", func() {
```

Comment:
It's analogous to what we have in mysql_ephemeral.go and I'm not sure if I can think of anything better, because the one thing that is covered by this tests really is the replication template (I was thinking of MySQL image, but that's not true, since we have mysql_ephemeral tests). I'm open to suggestions though.